### ISSUE TITLE
App should keep screen awake during uploading

### ISSUE THREAD

**Issue by melyux** [nextcloud/ios#1767]
Since the app can't upload in the background, it should at least keep the display of the iPhone on while it's actively uploading/downloading something to avoid the phone going to sleep. This should be pretty simple to implement.

**Comment by onatbas** [comment]
I agree. This feature is available in all the cloud storage client applications. OneDrive dims the screen and keeps the screen running.

**Comment by pug306d** [comment]
I have ~40,000 photos in my transfer queue and due to corporate policies the maximum time the phone will stay unlocked is 5mins so clearing that queue is very difficult without the feature for the app to prevent the phone from locking / sleeping

**Comment by stefanschaedeli** [comment]
+1

**Comment by eharris** [comment]
I very much agree with this, as like @pug306d it is very painful to upload a large amount of images without having to go change settings to temporarily avoid the screen lock.  Although I think it should only do so when it is plugged into power in order to avoid running down the battery if someone accidentally forgets to lock their screen before putting it back in their pocket while uploads are in progress.

**Comment by MondoGao** [comment]
We should add a feature called focus backup that works just like Synology Photos
